### DISCUSSION TITLE
MetaData error : 'metadata/generated_with' is not used for content. The field will be ignored.

### DISCUSSION THREAD

**Discussion by RavoGames** [bridge-core/editor#1055] [Q&A] [answered]
Hi, im not exactly sure how to use GitHub but I've been trying to use a mod I made and this error comes up (In Minecraft) and every time I load in into a Minecraft world with the mod the mod doesn't load. The error is as follows: (The property '/metadata/generated_with' is not used for this type of content. The field will be ignored.) Ive spent aver 2 hours trying to fix it.
![image](https://github.com/bridge-core/editor/assets/156178858/ab493473-2ebd-490e-9538-a501f2edec6e)

**Comment by outercloudstudio** [comment, marked as answer]
This warning is harmless, however if you'd like to get rid of it you can uncheck this setting.![Screenshot_20240110-071953.png](https://github.com/bridge-core/editor/assets/60296487/05583a95-1d5e-4e6d-9fca-1f3723233fa2)

**Comment by RavoGames** [comment]
Thank You so much, That got rid of the error, but the files still won't be imported into the world.
![image](https://github.com/bridge-core/editor/assets/156178858/338f87cd-81a4-4729-b8fd-ff451e1da47e)

**Reply by outercloudstudio** [reply]
Interesting. You should join the bridge discord and post a message in addon help. Unfortunately, I forgot how to fix that error.

**Comment by RavoGames** [comment]
Ok, well thanks for the help thus far

**Comment by TrashBagStudios** [comment]
How specifically do I fix the error, I have already turned that setting off and I have already synced the bridge. It still says the error,

**Reply by outercloudstudio** [reply]
Try restarting watch mode

**Reply by TrashBagStudios** [reply]
it didn't work but after that it took away the ID error that was alongside it.
So thank you for solving a problem :)

**Reply by Ralfs88** [reply]
just say no to the error. Embrace your power, and say NO.
